presto-main/src/test/java/com/facebook/presto/operator/scalar/TestPageProcessorCompiler.java

```diff
@@ -62,7 +63,7 @@ public void testNoCaching()
             throws Throwable
     {
         MetadataManager metadata = METADATA_MANAGER;
```

Comment:
only one test is using this static `METADATA_MANAGER`, to be consistent with the other tests we can remove this and create a new instance with before/after methods (see below).